docs/antrea-network-policy.md

```diff
@@ -1102,6 +1107,47 @@ spec:
       - fqdn: "svcA.default.svc.cluster.local"
 ```
 
+## Node Selector
+
+NodeSelector selects certain Nodes which match the label selector.
+When used in the `to` field of an egress rule, it adds the Node IPs to the rule's destination address group;
+when used in the `from` field of an ingress rule, it adds the Node IPs to the rule's source address group.
+
+Notice that when a rule with a nodeSelector applies to a Node, it only restricts the traffic to/from certain IPs of the Node.
+The IPs include:  
+
+1. The Node IP (the IP address in the Node API object)
+2. The Antrea gateway IP (the IP address of the interface antrea-agent will create and use for Node-to-Pod communication)
+3. The transport IP (the IP address of the interface used for tunneling or routing the traffic across Nodes) if it's different from Node IP
+
+Traffic to/from other IPs of the Node will be ignored.
+Meanwhile, `NodeSelector` doesn’t affect the traffic from Node to Pods running on that Node. Such traffic will always be allowed to make sure that [agents on a Node (e.g. system daemons, kubelet) can communicate with all Pods on that Node](https://kubernetes.io/docs/concepts/services-networking/#the-kubernetes-network-model) to perform liveness and readiness probes. For more information, see [https://github.com/antrea-io/antrea/pull/104](https://github.com/antrea-io/antrea/pull/104).
```

Comment:
Please break the line into multiple lines, aligning with other lines

Reply:
Updated.